org_status/formatters/__init__.py

```diff
@@ -0,0 +1,15 @@
+class RepoListFormatter:
+    NAME = None
+
+    def convert_repo_list_to_format(self, repos):
```

Comment:
We should have a better name and I prefer if we still retain the encode, decode terminology. Maybe call this `encode_repo_list()`?